src/modules/imageresizer/ui/Properties/Settings.settings

```diff
@@ -1,84 +0,0 @@
-<?xml version='1.0' encoding='utf-8'?>
```

Comment:
There is currently one setting that is being stored in registry in the Settings.cpp file which is the Enable/Disable ImageResizer option. It is currently implemented to mimic how PowerRename does the same. This settings value is read whenever a context menu is opened on a file so there could be performance overhead of reading a file everytime a context menu is triggered. This could be migrated when #1206 and the WinUI settings migration is done.